templates/tarsnap.key

```diff
@@ -0,0 +1 @@
+{{ TARSNAP_KEY }}
```

Comment:
Thanks. [Done here](https://github.com/open-craft/ansible-backup-to-tarsnap/pull/1/files#diff-2444ad0870f91f17ca6c2a5e96b26823R9)